File: app/src/main/java/com/zxcx/zhizhe/ui/card/hot/HotCardContract.java
package com.zxcx.zhizhe.ui.card.hot;

import com.zxcx.zhizhe.mvpBase.GetView;
import com.zxcx.zhizhe.mvpBase.IGetPresenter;

import java.util.List;

public interface HotCardContract {

    interface View extends GetView<List<CardBean>> {
    }

    interface Presenter extends IGetPresenter<List<CardBean>> {
    }
}

File: app/src/main/java/com/zxcx/zhizhe/ui/card/hot/HotCardContract.kt
package com.zxcx.zhizhe.ui.card.hot

import com.zxcx.zhizhe.mvpBase.GetView
import com.zxcx.zhizhe.mvpBase.IGetPresenter
import com.zxcx.zhizhe.ui.my.invite.InviteBean
import com.zxcx.zhizhe.ui.welcome.ADBean

interface HotCardContract {

    interface View : GetView<MutableList<CardBean>> {
        fun inputInvitationCodeSuccess(bean: InviteBean)
        fun getADSuccess(list: MutableList<ADBean>)
        fun errormsg(msg:String)
    }

    interface Presenter : IGetPresenter<MutableList<CardBean>> {
        fun inputInvitationCodeSuccess(bean:InviteBean)
        fun getADSuccess(list: MutableList<ADBean>)
        fun errormsg(msg:String)
    }
}

